Makefile

```diff
@@ -127,18 +127,35 @@ build: airflow-build
 
 build-dev: airflow-dev-build
 
+clean:
+	$(DOCKER_COMPOSE) down -v
+
+airflow-initdb:
+	$(DOCKER_COMPOSE) run --rm  webserver initdb
+
+
+end2end-test:
+	$(MAKE) clean
+	$(MAKE) airflow-initdb
+	$(DOCKER_COMPOSE) run --rm  test-client
+	$(MAKE) clean
+
 ci-build-dev:
 	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" build-dev
 
 ci-test-exclude-e2e: build-dev
-	$(DOCKER_COMPOSE_CI) run --rm peerscout-dags-dev ./run_test.sh
+	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" \
+	 run --rm peerscout-dags-dev ./run_test.sh
 
 ci-test-including-end2end: build-dev
-	$(DOCKER_COMPOSE_CI) run --rm  test-client
-	$(DOCKER_COMPOSE_CI) down -v
+	$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" \
+	 run --rm  test-client
```

Comment:
I think this line is causing it to fail. We want to delegate to `end2end-test`.
i.e. `$(MAKE) DOCKER_COMPOSE="$(DOCKER_COMPOSE_CI)" end2end-test`